core-blocks/paragraph/index.js

```diff
@@ -366,7 +351,7 @@ export const settings = {
 
 				const textClass = getColorClass( 'color', textColor );
 				const backgroundClass = getColorClass( 'background-color', backgroundColor );
-				const fontSizeClass = fontSize && FONT_SIZES[ fontSize ] && `is-${ fontSize }-text`;
+				const fontSizeClass = fontSize && `is-${ fontSize }-text`;
```

Comment:
Is it safe to remove this check?

Reply:
Yes, it is safe to remove this check. And I think, we should remove it. If we have set of font sizes now, and later we change the set we should probably keep using the same classes as before we should not remove the class just because we changed the set of FONT_SIZES. This will be changed again in https://github.com/WordPress/gutenberg/pull/6628 when this logic is abstracted and removed from the paragraph.